package.json

```diff
@@ -53,15 +53,19 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.18.10",
+    "@babel/preset-env": "^7.18.10",
     "@types/react": "^18.0.14",
     "@types/react-dom": "^18.0.5",
     "assert": "^2.0.0",
+    "babel-jest": "^28.1.3",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
     "cypress": "^10.3.1",
     "eslint-plugin-cypress": "^2.12.1",
     "ethers": "^5.6.9",
     "https-browserify": "^1.0.0",
+    "jest": "^28.1.3",
```

Comment:
Jest comes with create-react-app, we don't have to install it separately

Reply:
yes, if install separately it it gives duplicated files, so it had been deleted.

Reply:
better to have clear dependency and add packages that you use in your app.